AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml

```diff
@@ -24,7 +24,15 @@
             @foreach (var item in Model) {
                 <tr>
                     <td>
+                        @if (String.IsNullOrEmpty(item.LogoUrl))
```

Comment:
I think the if statement should be inside the a tag.

Also don't forget a the alt attribute on the logo `alt="@item.Name"`